libsolidity/interface/ABI.cpp

```diff
@@ -36,7 +36,8 @@ Json::Value ABI::generate(ContractDefinition const& _contractDef)
 		Json::Value method;
 		method["type"] = "function";
 		method["name"] = it.second->declaration().name();
-		method["constant"] = it.second->isConstant();
+		// FIXME: constant should be removed at the next breaking release
+		method["constant"] = it.second->isView();
```

Comment:
Basically it is only used here and in one test. Probably worth replacing with a direct check to statemutability.